src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+
+    private final ColorSupplier colorSupplier;
+    private final Random random;
+
+    public Lottery(ColorSupplier colorSupplier, Random random) {
+        this.colorSupplier = colorSupplier;
+        this.random = random;
+    }
```

Comment:
Look, dependency injection is a very good practice, no doubt. Setting services you rely on via constructor is a good thing and will be automated further in the course.
But code should be consistent. If you use such solution for Lottery then it should be same approach for ColorSupplier.
At the current stage you may choose if you wish to use approach with constructor and make similar constructor for ColorSupplier class OR you may use approach from ColorSupplier for lottery and initialize class variables with 'new' object during decalaration